src/oic/oic/__init__.py

```diff
@@ -268,9 +268,12 @@ def add_token(self, resp):
 }
 
 
-def response_types_to_grant_types(response_types):
+def response_types_to_grant_types(response_types, kwargs):
```

Comment:
kwargs is unusual as a single argument, python style would be **kwargs.

probably rename it to extra_grant_types